app/models/product.rb

```diff
@@ -10,10 +10,11 @@ class Product < ApplicationRecord
   validates :gender,      presence: true
   validates :quantity,    presence: true
 
-
   enum gender: {
     "Female": 0,
     "Male": 1
   }
 
+  scope :premium, -> { where("price > ?", 3000) }
```

Comment:
Hey Ash! Thanks for the review! Yup I think it's needed. Or at least, cause (price > 3000) didn't work so I went googling and found this solution. I think the ? is needed because it's a range of values instead of a straightforward query like (price: 3000).